package.json

```diff
@@ -56,5 +56,8 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
+    },
+    "dependencies": {
```

Comment:
why this change? It's repo specific entry, not TS module specific one

Reply:
Sorry - added that for running tests. I'll drop it.